shared/utils/env_utils.py

```diff
@@ -171,3 +172,77 @@ def pawns_around(board: Board, pawn: tuple, distance: int):
             if (x+i, y+j) in board.get_black_coordinates():
                 count += 1
     return count
+
+def piece_parser(piece: Piece) -> int:
+    """
+    Return the index of the boolean array (which represents the board) used as a input for the policy network of the DQN
+
+    Arg:
+    Piece object
+
+    Example:
+    If the piece given is the KING, the function will return 1
+    The second array given as input will be the one displaying the position of the KING in the 9x9 board (index 1 means second element)
+    """
+    state_pieces = {Piece.DEFENDER : 0,
+                    Piece.KING : 1,
+                    Piece.ATTACKER : 2,
+                    Piece.EMPTY : 3,
```

Comment:
Piece.Empty shouldn't be tracked. Remove it and if a piece.empty is passed, raise a Value Error